images/linux/scripts/installers/runner-package.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+################################################################################
+##  File:  runner-package.sh
+##  Desc:  Downloads and Installs runner package
+################################################################################
+
+# Source the helpers for use with the script
+source $HELPER_SCRIPTS/install.sh
+
+LATEST_VERSION = $(
```

Comment:
we should use the `get_github_package_download_url` helper here

Reply:
Would it be something like: `downloadUrl=$(get_github_package_download_url "actions/runner" "contains(\"actions-runner-linux-x64-\")" "latest")`